### ISSUE TITLE
find

### ISSUE THREAD

**Issue by honnycy** [LavaLite/cms#259]

**Comment by huntr-helper** [comment]
### Bug Bounty

We have opened up a bounty for this issue on our bug bounty platform. Want to solve this vulnerability and get rewarded 💰? Go to https://huntr.dev/

We will submit a pull request directly to your repository with the fix as soon as possible. Want to learn more? Go to https://github.com/418sec/huntr 📚

_Automatically generated by @huntr-helper..._

**Comment by alromh87** [comment]
Can someone confirm this issue is still valid, and if so which version of the code your using

**Comment by B3EF** [comment]
you are right @alromh87 its not a valid code, as i was testing lavalite in the last week i didn't saw a client interface like this.

**Comment by honnycy** [comment]
@alromh87 i test it in LavaLite - 5.5
